### PR TITLE
Integrate new web3.storage. Closes #141

### LINKED ISSUE #141: Migrate to the new web3.storage API
See https://mailchi.mp/web3.storage/major-changes-to-web3storage?e=77416530cd

> what this means for you, a current web3.storage user. Here’s the tl;dr:
> - The new upload API and client is live. Its account system is completely separate from that of the legacy web3.storage.
> - The legacy web3.storage upload API, pinning service API, and w3name will sunset on January 9, 2024. Until then, you will still be able to upload to your legacy account up to your free account limit. The old dashboard can be found [here](https://old.web3.storage/account/).
> - All uploads to the legacy web3.storage prior to January 9, 2024 will continue to be stored until further notice, though over time latency and availability may degrade.

/cc @juliangruber FYI

### PR DESCRIPTION
**Pull request by juliangruber**
- [x] Matching `spark-evaluate` PR https://github.com/filecoin-station/spark-evaluate/pull/70

Files are now published to the root entry of the CID (vs at `/measurements.json`), therefore this PR and the `spark-evaluate` PR need to be merged together.

Closes #141

### CODE REVIEW

File: spark-publish/index.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const publish = async ({
     'measurements.json',
     { type: 'application/json' }
   )
-  const cid = await web3Storage.put([file])
+  const cid = await web3Storage.uploadFile(file)
```
Comment (juliangruber) [inline spark-publish/index.js:56]: I noticed a change in behavior here, see https://filecoinproject.slack.com/archives/C027XP5BTGB/p1700818334024649

Comment (juliangruber) [reply spark-publish/index.js:56]: This new one is actually simpler, since we don't need to support the extra complexity of unixfs

**Review by bajtos — APPROVED** [review]

**Comment by juliangruber** [comment]
> Can you please grep through the source code and remove any remaining places that may be using `WEB3_STORAGE_API_TOKEN`?

There are no string matches for this, were you thinking about something particular?

**Comment by bajtos** [comment]
> > Can you please grep through the source code and remove any remaining places that may be using `WEB3_STORAGE_API_TOKEN`?
> 
> There are no string matches for this, were you thinking about something particular?

Not really. Just to make sure we don't accidentally leave behind some dead code. Thanks for checking!

**Pull request merged**